doc/doxygen/scripts/program2doxygen

```diff
@@ -43,8 +44,21 @@ do {
 
     s/\t/        /g;
 
+    # We are entering code fragments
+    if (($state != $code_fragment_mode) && m!^\s*//\s*\@code!)
+    {
+        # Cleanly close @code segments in program mode:
+        if ($state == $program_mode)
+        {
+            print " * \@endcode\n";
+            print " * \n";
+        }
+        $state = $code_fragment_mode;
+        # Make sure we distinguish from the other code style
+        print " * <div class=inlineddealii>\n";
```

Comment:
maybe call it `CodeFragmentInTutorialComment` to make its purpose clearer?

Reply:
and don't you have to close the `</div>` somewhere?

Reply:
it is closed in line 148: https://github.com/dealii/dealii/pull/6567/files#diff-b7c2f9c8d8a4164bbc4d934b8f65059fR148